metrics/metrics.go

```diff
@@ -132,6 +134,16 @@ func NewMetrics() *CatalystAPIMetrics {
 			Help:    "Time taken for catabalancer load balancing requests",
 			Buckets: []float64{.005, .01, .025, .05, .1, .25, .5, 1, 2.5, 5, 10},
 		}, []string{"success", "request_type", "mist_match", "background"}),
+		CatabalancerSendMetricDurationSec: promauto.NewHistogram(prometheus.HistogramOpts{
+			Name:    "catabalancer_send_metric_duration",
+			Help:    "Time taken to send catabalancer node metric updates",
+			Buckets: []float64{.005, .01, .025, .05, .1, .25, .5, 1, 2.5, 5, 10},
+		}),
+		CatabalancerSendDBDurationSec: promauto.NewHistogramVec(prometheus.HistogramOpts{
+			Name:    "catabalancer_send_db_duration",
+			Help:    "Time taken to send catabalancer node metric updates to DB",
```

Comment:
This one is for sending metric update to DB.
What is `catabalancer_send_metric_duration` about? It includes the time for pulling the metrics from `mist`, right?
I'm not sure if I understand the difference reading the `Help:` message

Reply:
That's right so we have a better idea of where any slowness lies, I'll update the help text